.github/workflows/pr-test.yml

```diff
@@ -161,19 +141,21 @@ jobs:
           name: build
           path: ${{ env.BUILD_OUT_ROOT }}
 
-      - uses: technote-space/get-diff-action@v6.1.0
-        with:
-          PATTERNS: files/**/*.+(png|jpeg|jpg|gif|svg|webp)
-          ABSOLUTE: true
-          SET_ENV_NAME: GIT_DIFF_FILES
+      - name: Get changed images
+        if: ${{ env.GIT_DIFF_CONTENT }}
+        run: |
+          DIFF_IMAGES=$(gh pr view ${{ github.event.number }} --json files --jq ".files.[].path" | egrep -i "^files/.*/.*\.(png|jpeg|jpg|gif|svg|webp)$" | tr "\n" " ")
```

Comment:
Could we retrieve **all** changed files in `files/` (`DIFF_FILES`) **once** (above near line 42) rather than twice, and then derive `DIFF_DOCUMENTS` and `DIFF_IMAGES` from it?